ui/src/app.js

```diff
@@ -2136,3 +2144,30 @@ function setReceiptPrint (receiptStatus, smsReceiptStatus) {
       break
   }
 }
+
+function setScreenOptions (opts) {
+  if (opts.rates) {
+    opts.rates.active ? $('#rates-section').show() : $('#rates-section').hide()
+  }
+}
```

Comment:
I don't know how/if it could happen, but imagine that `opts.rates.active` is true in an iteration, and on the next iteration there's no `opts.rates`. The rates won't be disabled though they should. Right?

```js
(opts.rates && opts.rates.active) ? $('#rates-section').show() : $('#rates-section').hide()
```